src/backend/metal/src/device.rs

```diff
@@ -44,6 +44,119 @@ const ARGUMENT_BUFFER_SUPPORT: &[MTLFeatureSet] = &[
     MTLFeatureSet::macOS_GPUFamily1_v3,
 ];
 
+const ASTC_PIXEL_FORMAT_FEATURES: &[MTLFeatureSet] = &[
+    MTLFeatureSet::iOS_GPUFamily2_v1,
```

Comment:
I don't think we need multiple versions of each family - only the base version (where support appeared) needs to be checked